python/paddle/fluid/dygraph/nn.py

```diff
@@ -1308,6 +1309,12 @@ def __init__(self,
             dtype=self._dtype)
         self._variance.stop_gradient = True
 
+        self._has_reserve_space = False
+        if data_layout == 'NHWC':
+            flag = os.environ.get('FLAGS_cudnn_batchnorm_spatial_persistent')
```

Comment:
Why not use paddle.get_flags instead of os.environ.get

Reply:
OK，I will fix it in next PR